funnel/views/project.py

```diff
@@ -89,17 +90,15 @@ class ProfileProjectView(ProfileViewMixin, UrlForView, ModelView):
     @requires_roles({'admin'})
     def new_project(self):
         form = ProjectForm(model=Project, parent=self.obj)
-        # Profile URLs:
-        # HasGeek: https://hasgeek.com/rootconf (no /)
-        # Talkfunnel: https://rootconf.talkfunnel.com/ (has /)
-        form.name.prefix = self.obj.url_for(_external=True)
-        if not form.name.prefix.endswith('/'):
-            form.name.prefix += '/'
+
         if request.method == 'GET':
             form.timezone.data = current_app.config.get('TIMEZONE')
         if form.validate_on_submit():
             project = Project(user=current_auth.user, profile=self.obj)
             form.populate_obj(project)
+            project.name = "{yyyymm}-{name}".format(
+                yyyymm=utcnow().strftime('%Y-%m'), name=make_name(project.title),
+            )
```

Comment:
Just realised we will need event date, not current date, but we don't have event date yet. I think we can default to using `project.make_name()` here.